hack/verify-codegen.sh

```diff
@@ -30,18 +35,23 @@ trap "cleanup" EXIT SIGINT
 
 cleanup
 
-# Save working tree state
 mkdir -p "${TMP_DIFFROOT}/pkg"
-cp -aR "${REPO_ROOT_DIR}/Gopkg.lock" "${REPO_ROOT_DIR}/vendor" "${TMP_DIFFROOT}"
+# Save working tree state
+for dir in ${DIRS_TOBE_INSPECTED[@]}; do
+  cp -aR "${REPO_ROOT_DIR}/${dir}" "${TMP_DIFFROOT}"
+done
 
 "${REPO_ROOT_DIR}/hack/update-codegen.sh"
```

Comment:
Could we run this in the temporary directory so we don't have to mess up the actual repo dir?

Reply:
done

Reply:
I meant, can we run `hack/update-codegen.sh` from the temporary directory instead of the repo directory? If not, that's fine, but if we run this on our own machines, it will overwrite local changes to our repo, report things are broken, but it will then be compliant, which is confusing user experience...

Reply:
This is the same across all repos, so far haven't seen any problems yet, so might just keep it

Reply:
k. Feel free to resolve